cmd/vic-init/main_linux.go

```diff
@@ -56,6 +57,9 @@ func main() {
 	extraconfig.Decode(src, &config)
 
 	debugLevel = config.Diagnostics.DebugLevel
+	if debugLevel > 1 {
```

Comment:
Actually I only did this for my debugging, and it should not have ended up in the PR :-) But if you think it is still a good idea for debug > 2, I'll keep it with that change.